adafruit_ble_file_transfer.py

```diff
@@ -117,13 +123,13 @@ class ProtocolError(BaseException):
 class FileTransferClient:
     """Helper class to communicating with a File Transfer server"""
 
-    def __init__(self, service):
+    def __init__(self, service) -> None:
         self._service = service
 
         if service.version < 3:
             raise RuntimeError("Service on other device too old")
 
-    def _write(self, buffer):
+    def _write(self, buffer) -> None:
```

Comment:
I think this one might be ReadableBuffer, but I'm not 100% sure, I'm still not confident I have the usages for them down correctly.

Reply:
You got it!